sdk/src/main/java/com/meniga/sdk/converters/MenigaRedemptionConverter.kt

```diff
@@ -0,0 +1,35 @@
+package com.meniga.sdk.converters
+
+import com.google.gson.Gson
+import com.meniga.sdk.helpers.type
+import com.meniga.sdk.models.offers.MenigaSimilarBrandSpendingDetails
+import com.meniga.sdk.models.offers.redemptions.MenigaRedemptions
+import com.meniga.sdk.models.offers.redemptions.MenigaScheduledReimbursement
+import okhttp3.ResponseBody
+import retrofit2.Converter
+import retrofit2.Retrofit
+import java.lang.reflect.Type
+
+/**
+ * Copyright 2017 Meniga Iceland Inc.
+ */
+class MenigaRedemptionConverter(private val gson: Gson) : MenigaConverter() {
```

Comment:
Is there a reason the converter is called MenigaRedemptionConverter but the only type it handles is MenigaSimilarBrandSpendingDetails?

Reply:
Good catch